metacat-client/src/main/java/com/netflix/metacat/client/module/MetacatErrorDecoder.java

```diff
@@ -51,7 +51,7 @@ public Exception decode(final String methodKey, final Response response) {
                 message = Util.toString(response.body().asReader());
                 try {
                     final ObjectNode body = METACAT_JSON.parseJsonObject(message);
-                    message = body.path("error").asText();
+                    message = body.path("message").asText();
```

Comment:
This is my only actual concern.  Unless you can get everyone to update to the latest client at the same time you release this version of Metacat, the existing clients will work with the new server they just won't see the error messages.